contracts/LockedTokens.cdc

```diff
@@ -181,21 +181,47 @@ pub contract LockedTokens {
         /// Registers a new node operator with the Flow Staking contract
         /// and commits an initial amount of locked tokens to stake
         pub fun registerNode(nodeInfo: StakingProxy.NodeInfo, amount: UFix64) {
+            if let nodeStaker <- self.nodeStaker <- nil {
+                let stakingInfo = FlowIDTableStaking.NodeInfo(nodeID: nodeStaker.id)
+                
+                assert(
+                    stakingInfo.tokensStaked + stakingInfo.totalTokensStaked + stakingInfo.tokensCommitted + stakingInfo.tokensUnstaking + stakingInfo.tokensUnstaked + stakingInfo.tokensRewarded == 0.0,
```

Comment:
yes, I could do that! I'll make the change

Reply:
Actually, does it make sense to make it a helper function if it is only used once? I feel like it would be better as a helper function in the actual `NodeInfo` or `DelegatorInfo` definitions in the staking contract. @robmyers @psiemens

Reply:
Yeah I agree, it'd be better if it was a method on those structs. Not worth adding it here if that's the better option